readme.md

```diff
@@ -225,3 +225,9 @@ As in: OOPs I used Java!
 **A:** Because he used up all his cache
 
 ---
+
+**Q:** Why did the developer have to quit smoking?
+
+**A:** Because they imposed a new syntax.
```

Comment:
```suggestion
**A:** Because they couldn't afford to pay the new syntax.
```

Reply:
Maybe even put (Sin Tax) in parenthesis so it's a little more clear to the reader

Reply:
Or maybe something like "because she heard there was a new syntax", which points out the homophone w/out having to make it obvious.